src/test/java/Homework17.java

```diff
@@ -35,8 +33,28 @@ public void enterPassword(String password) {
         passwordField.click();
         passwordField.sendKeys(password);
         }
-        public void clickSubmit() {
-        WebElement submitButton = driver.findElement(By.cssSelector("button[type='submit']"));
-        submitButton.click();
+        public void allSongs() throws InterruptedException {
+        WebElement allSongs = driver.findElement(By.cssSelector("#sidebar > section.music > ul > li:nth-child(3) > a"));
+        allSongs.click();
+        Thread.sleep(2000);
+        }
+        public void songName() throws InterruptedException {
+            WebElement songName = driver.findElement(By.xpath("//*[@id=\"songsWrapper\"]/div/div/div[1]/table/tr[1]/td[2]"));
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

_For example, here is the locator (Copy > Copy Xpath):_  `//*[@id="sidebar"]/section[1]/ul/li[3]/a`

**BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.**
For example, here is the correct locator: ` //a[@class='songs']`